### ISSUE TITLE
Page should contain a level-one heading

### ISSUE THREAD

**Issue by christinach** [pulibrary/static-tables#103]
### Steps to recreate
Go to https://static-tables-staging.princeton.edu/
Open the dev console.
Scan the page using axe-core tools browser extension

### URL
https://static-tables-staging.princeton.edu/

### Issues
Page must have a level-one heading

### Severity
High

### WCAG Success Criterion
[S.C 2.4.10](https://www.w3.org/WAI/WCAG22/Understanding/section-headings) AAA
[S.C. 1.3.1](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships) A

### WCAG Conformance level (A, AA, AAA)
A
AAA

### User impact
AAA Intent: The intent of this Success Criterion is to provide headings for sections of a Web page, when the page is organized into sections. For instance, long documents are often divided into a variety of chapters, chapters have subtopics, etc. When such sections exist, they need to have headings that introduce them. This clearly indicates the organization of the content, facilitates navigation within the content, and provides mental "handles" that aid in comprehension of the content. Other page elements may complement headings to improve presentation (e.g., horizontal rules and boxes), but visual presentation is not sufficient to identify document sections.

This provision is included at Level AAA because it cannot be applied to all types of content and it may not always be possible to insert headings. For example, when posting a pre-existing document to the Web, headings that an author did not include in the original document cannot be inserted. Or, a long letter would often cover different topics, but putting headings into a letter would be very strange. However, if a document can be broken up into sections with headings, it facilitates both understanding and navigation. 

AAA benefits: People who are blind will know when they have moved from one section of a Web page to another and will know the purpose of each section.
People with some learning disabilities will be able to use the headings to understand the overall organization of the page content more easily.
People who navigate content by keyboard will be able to jump the focus from heading to heading, enabling them to find quickly content of interest.
In pages where content in part of the page updates, headings can be used to quickly access updated content.

### Recommended fix
Page must have a level-one heading
Element location: `<html>`

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
[Accessibility audit_05152024_Static-Tables](https://docs.google.com/spreadsheets/d/1_-Khj5TezyEu2gVb2jJczWCUt04bt5_P8-zL0f269kQ/edit#gid=0)

The <h1> describes the page as a whole (and should be similar to the page <title>). A page should typically have only one <h1>. ([WebAIM Headings](https://webaim.org/techniques/semanticstructure/#headings))
